tutorials/fileToJdbc/pom.xml

```diff
@@ -306,27 +283,17 @@
       <artifactId>super-csv</artifactId>
       <version>2.4.0</version>
     </dependency>
-    <!--<dependency>-->
-      <!--<groupId>com.datatorrent</groupId>-->
-      <!--<artifactId>malhar-library</artifactId>-->
-      <!--<version>3.5.0-SNAPSHOT</version>-->
-      <!--<scope>test</scope>-->
-    <!--</dependency>-->
-      <dependency>
-          <groupId>com.datatorrent</groupId>
-          <artifactId>dt-contrib</artifactId>
-          <version>3.1.1</version>
-          <exclusions>
-              <exclusion>
-                  <groupId>*</groupId>
-                  <artifactId>*</artifactId>
-              </exclusion>
-          </exclusions>
-      </dependency>
-      <!--<dependency>-->
-          <!--<groupId>org.apache.apex</groupId>-->
-          <!--<artifactId>malhar-library</artifactId>-->
-          <!--<version>3.4.0</version>-->
-      <!--</dependency>-->
+    <dependency>
```

Comment:
Not sure why you need this dependency; it actually causes the build to fail for me:
```
[ERROR] Failed to execute goal on project FileToJdbcApp: Could not resolve dependencies for project com.example:FileToJdbcApp:jar:1.0-SNAPSHOT: Could not find artifact com.datatorrent:dt-contrib:jar:3.1.1 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
```

After removing it, the build is OK.